swo_aws_extension/flows/steps/setup_parameters.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from copy import deepcopy
+
+from mpt_extension_sdk.flows.pipeline import Step
+from mpt_extension_sdk.mpt_http.base import MPTClient
+from mpt_extension_sdk.mpt_http.mpt import update_order
+
+from swo_aws_extension.flows.order import InitialAWSContext
+
+logger = logging.getLogger(__name__)
+
+
+class SetParametersVisibleStep(Step):
+    def set_hidden_paramter(self, parameter, hidden=False):
+        """
+        Update the visibility of the parameter to read-only.
+
+        Args:
+            parameter (dict): The parameter to update.
+        """
+        new_parameter = deepcopy(parameter)
+        if "constraints" not in new_parameter:
```

Comment:
I think `constraints` is a required field in the parameter definition, isn't it?